### PR TITLE
chore: version bump

### PR DESCRIPTION
**Pull request by khatruong2009**
*Issue #, if available:*

*Description of changes:*
- Version Bump

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Jordan-Nelson — APPROVED** [review]

**Review by Equartey — APPROVED** [review]

File: packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml
Comment (NikaHsn) [inline packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml:1]: is this missing version bump and changelog update?

File: packages/auth/amplify_auth_cognito_dart/pubspec.yaml
Comment (NikaHsn) [inline packages/auth/amplify_auth_cognito_dart/pubspec.yaml:1]: is this missing version bump and changelog?

File: packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml
Comment (Jordan-Nelson) [reply packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml:1]: It was bumped here: https://github.com/aws-amplify/amplify-flutter/pull/4001/files#diff-ae45f440f37b25dfcf713bafe1894228cb24c6cebad898abb61e7571fbec5a49R3

File: packages/auth/amplify_auth_cognito_dart/pubspec.yaml
Comment (Jordan-Nelson) [reply packages/auth/amplify_auth_cognito_dart/pubspec.yaml:1]: https://github.com/aws-amplify/amplify-flutter/pull/4001/files#diff-418ce1229065cdd1d750534edfbd8804ec2d25f3b1cbffcfa9b68a925c6ab816R3

**Pull request merged**